index.html

```diff
@@ -113,6 +113,32 @@ <h2>A project for first-time contributions</h2>
             </ul>
           </div>
         </div>
+        <!-- ________ *TEMPLATE* Contributor card END ________  -->
+          <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Soumojit</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://x.com/home" target="_blank">Soumojit</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Soumojit-007" target="_blank">Soumojit-007</a>
+          </p>
+          <p class="about">about you</p>
```

Comment:
Write something about you
e.g.
```suggestion
          <p class="about">Hi, I am Soumojit. I am from ... . Currently, I am studying ... / working on ... .</p>
```